randovania/games/am2r/pickup_database/pickup-database.json

```diff
@@ -232,7 +267,10 @@
             "pickup_category": "beam",
             "broad_category": "beam_related",
             "model_name": "Ice Beam",
-            "offworld_models": {},
+            "offworld_models": {
+                "prime1": "Ice Beam",
+                "dread": "powerup_icemissile"
```

Comment:
I think it's better to have no model than to use ice missiles for ice beam tbh, but that one's up to you

Reply:
Every other beam has an equivalent in dread right now, so even though ice missiles might not be a 100% perfect fit, it's close enough for me